testing/fixtures/command_unset_fixtures.go

```diff
@@ -92,3 +94,65 @@ func unsetShouldFailWhenConfigurationFileIsInInvalidFormat(feature string) Test
 		},
 	}
 }
+
+func shouldUnsetMultipleCommands(feature string) Test {
```

Comment:
Let's also have a test for comments not being empty after removing one of the commands.

ex.
```yaml
project: Sample Project
commands:
  - build: go build
  - test: go test
  - lint: go lint
```

```bash
1build unset test missing1 build missing2 
```

After command:
```yaml
project: Sample Project
commands:
  - test: go test
```

AFAIK, we are not testing this flow.

Reply:
Yep that sounds good, I will add this test case in.

But to clarify, in the example the final output should be the following right? Since build and test is unset.

```
project: Sample Project
commands:
  - test: go test
```

Reply:
Yes. you are right. Updated comment.